view/form/PublishForm.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace oat\taoPublishing\view\form;
+
+class PublishForm extends \tao_helpers_form_FormContainer
+{
+
+    /**
+     * @inheritDoc
+     * @throws \common_Exception
+     * @throws \Exception
+     */
+    protected function initForm()
+    {
+        $this->form = new \tao_helpers_form_xhtml_Form('publish');
+
+        $createElt = \tao_helpers_form_FormFactory::getElement('save', 'Free');
+        $createElt->setValue(
+            '<button class="form-submitter btn-success small" type="button"><span class="icon-publish"></span> '
+            . __('Publish') . '</button>'
+        );
+        $this->form->setDecorators([
+            'actions-bottom' => new \tao_helpers_form_xhtml_TagWrapper(['tag' => 'div', 'cssClass' => 'form-toolbar']),
+         ]);
+        $this->form->setActions([], 'top');
+        $this->form->setActions([$createElt], 'bottom');
+    }
+
+    /**
+     * @inheritDoc
+     */
+    protected function initElements()
+    {
+        // TODO: Implement initElements() method.
```

Comment:
This will be done on another task or not needed at all? If no init is necessary, please remove TODO